config/initializers/jbuilder_prettify.rb

```diff
@@ -0,0 +1,26 @@
+require "jbuilder"
+
+class Jbuilder
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @attributes [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)
* [InstanceVariableAssumption] assumes too much for instance variable @prettify [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)
* [PrimaDonnaMethod] has prima donna method `prettify!` [:ref](https://github.com/troessner/reek/blob/master/docs/Prima-Donna-Method.md)
* [PrimaDonnaMethod] has prima donna method `target!` [:ref](https://github.com/troessner/reek/blob/master/docs/Prima-Donna-Method.md)

<!-- Comment by [SideCI](https://sideci.com) -->